components/brave_ads/browser/notification_helper_android.cc

```diff
@@ -73,13 +72,14 @@ NotificationHelper* NotificationHelper::GetInstanceImpl() {
 
 ///////////////////////////////////////////////////////////////////////////////
 
-bool NotificationHelperAndroid::IsBraveAdsNotificationChannelEnabled() const {
+bool NotificationHelperAndroid::IsBraveAdsNotificationChannelEnabled(bool foregroundChannel) const {
```

Comment:
Can we please change to snake_case

Reply:
nit: > 80 line length, can you please run changes through the linter

Reply:
done